### ISSUE TITLE
Accessibility: Collapsed inline editable form fields can be tabbed to

### ISSUE THREAD

**Issue by GuySartorelli** [dnadesign/silverstripe-elemental#1374]
After expanding a block and then collapsing it again, all of the form fields in the (hidden) element form are accessible by tabbing to them. Instead, they should be skipped over since they are hidden.

Note: Since this is a bug for existing functionality, it should be fixed in patches where appropriate.

## New issues created
- https://github.com/silverstripe/silverstripe-elemental/issues/1378 (this issue originally had 2x bugs on it, moved one of the bugs to this new issue)

## Kitchen sink CI
- https://github.com/creative-commoners/recipe-kitchen-sink/actions/runs/18302848952

## PRs (merge top to bottom)
- https://github.com/silverstripe/silverstripe-behat-extension/pull/312
- https://github.com/silverstripe/silverstripe-admin/pull/2011
- https://github.com/silverstripe/silverstripe-asset-admin/pull/1615
- https://github.com/silverstripe/silverstripe-linkfield/pull/413
- https://github.com/silverstripe/silverstripe-elemental/pull/1380

**Comment by emteknetnz** [comment]
Note that we cannot use just `display:none` to remove the inputs from the accessibility tree, as one of the parent elements of the form will have this class on in (set in Content.js) `.element-editor-editform--rendered-not-visible`

InlineEditForm.scss has this note on the class

```css
  // Render the form off the screen so that it's not visible
  // Note that using display:none will mean the form is not rendered at all and cannot be submitted
  &--rendered-not-visible {
    position: absolute;
    top: -9999px;
    left: -9999px;
  }
```

This is important so that when the page is published, all inline editable element forms (which were loaded i.e. expanded at some point) are also submitted, including those have been collapsed

This probably only leaves setting `tabindex="-1"` on each input as a solution. I'm not sure how hard this will be as we are using redux-form, though we are adding our own form field components.

**Comment by GuySartorelli** [comment]
PRs merged
